lib/whitehall.rb

```diff
@@ -132,10 +132,6 @@ def self.admin_root
     @admin_root ||= Plek.new.external_url_for("whitehall-admin")
   end
 
-  def self.secrets
```

Comment:
Yes, looks like it's no longer used. It was added in https://github.com/alphagov/whitehall/commit/3be3fd649c366832f4210eb32c70d72b41592eba